README.md

```diff
@@ -202,6 +202,14 @@ Once installed set the files as such:
 Then to make sure prettier is run correctly, modify the on save actions:
 ![Intellij javascript onsave](intellij/intellij-javscript-actionsonsave.png)
 
+For Scala:
+
+Go to `Preferences/Settings > Editor > Code Style > Scala > Imports` and update:
+* Class count to use import with '_' -> 999
+* Ensure `Merge imports with the same prefix into one statement` is checked
+
+Then go to `Preferences/Settings > Editor > General > Auto import` and uncheck "Optimize imports on the fly" for Scala. This is necessary because there are some imports that are not explicitly used in the code but are necessary for the code to compile.
+
```

Comment:
I just added the settings that I know are relevant to what we want to brain. There are other things that are checked and I could not see a difference in the behavior of the autoimport, so I did not mention them here, I guess we can update this if we see in the future new required settings